features/record-detail/components/swim-stats-item.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@/styled-system/css';
+import { flex } from '@/styled-system/patterns';
+
+export const SwimStatsItem = () => {
+  return (
+    <div className={containerStyle}>
+      <h1 className={text.type}>자유형</h1>
+      <h2 className={text.distance}>1500m</h2>
+      <p className={text.lap}>124lap</p>
+    </div>
+  );
+};
+
+const containerStyle = flex({
+  gap: '2px',
+  flexDirection: 'column',
+  w: 'full',
+});
+
+// TODO: swim type에 따른 색상 변경
+const text = {
+  type: css({
+    textStyle: 'label1.normal',
+    fontWeight: 'medium',
+    color: 'primary.swim.자유형.default',
+  }),
```

Comment:
영법에 따라 색상만 바뀌는거라면 cva를 활용해서 색깔만 조정할 수 있도록 하면 좋을 것 같아요~!!

```suggestion 

stroke: cva({
    base: { textStyle: 'label1.normal', fontWeight: 'medium' },
    variants: {
      type: { 자유형: { color: 'primary.swim.자유형.default' }, 
      ...추가되는 영법 type },
    },
  }),
```

`className={css(text.stroke.raw({ type: '자유형' }))}` 로 스타일 부여